src/dashboard/src/components/filesystem_ajax/views.py

```diff
@@ -813,3 +815,25 @@ def download_fs(request):
     except ValueError:
         raise django.http.Http404
 
+def download(request, uuid):
```

Comment:
Updated the name and added a docstring.

I'm thinking `download_ss` should probably stream the response as well, since the bug I added streaming for likely affects some current Archivematica installations.